tests/functional/olp-cpp-sdk-dataservice-read/ApiTest.cpp

```diff
@@ -39,6 +37,7 @@
 #include <olp/core/porting/make_unique.h>
 
 #include "ApiClientLookup.h"
+#include "TokenEndpoint.h"
```

Comment:
Why is TokenEndpoint needed? Can you please have a look and change the code if needed.

Reply:
removed